src/topp/PercolatorAdapter.cpp

```diff
@@ -416,7 +417,12 @@ class PercolatorAdapter :
         
         double calc_mass = hit.getSequence().getMonoWeight(Residue::Full, charge)/charge;
         hit.setMetaValue("CalcMass", calc_mass);
-        
+
+        if (hit.metaValueExists("IsotopeError"))
+        {
+          float isoErr = stof(hit.getMetaValue("IsotopeError").toString());
```

Comment:
stof uses the current locale and might thus not work for files produced on a system with a different locale.
use the toString().toFloat() instead (if toFloat() doesn't work)